src/schema/show.ts

```diff
@@ -43,6 +43,15 @@ import EventStatus from "./input_fields/event_status"
 import { LOCAL_DISCOVERY_RADIUS_KM } from "./city/constants"
 import { ResolverContext } from "types/graphql"
 
+const ShowsFollowedContentType = new GraphQLObjectType<any, ResolverContext>({
+  name: "ShowsFollowedContent",
+  fields: () => ({
+    artists: {
+      type: new GraphQLList(Artist.type),
```

Comment:
should this be a connection?

Reply:
i pretty much followed the current implementation in Fair 
https://github.com/artsy/metaphysics/blob/1c1145abd24a3e2c572d672a88dab57d50d4d94f/src/schema/fair.ts#L38
but we can make it a connection here, what do you think?

Reply:
I don't mean to intrude or do 'drive by' code reviews (and am mindful of a rush for LD work). Connections have advantages over simple lists for several reasons: there's a schema for additional metadata (like pagination info, total counts, etc- which all frequently wind up being needed by the front-end at some point). Also, there are several Relay capabilities (used in Emission, pagination containers, etc.)- that essentially require a connection type and something conforming to that spec.

So it's just an automatic reflex- if I see a list/array of objects like this, I try to ask if it should be a connection! 😄 Totally up to you though.

Reply:
makes total sense, let me see if i can quickly add it 🙌

Reply:
@mzikherman Here’s a doc to point to for your next drive-by https://github.com/artsy/README/blob/master/playbooks/graphql-schema-design.md#connections-over-lists (and please do expand on it with your favourite arguments)